Formula/h/httpd.rb

```diff
@@ -31,7 +31,7 @@ class Httpd < Formula
   def install
     # fixup prefix references in favour of opt_prefix references
     inreplace "Makefile.in",
-      '#@@ServerRoot@@#$(prefix)#', '#@@ServerRoot@@'"##{opt_prefix}#"
+      '#@@ServerRoot@@#$(prefix)#', '#@@ServerRoot@@' + "##{opt_prefix}#"
```

Comment:
It's probably gonna want:

```suggestion
      '#@@ServerRoot@@#$(prefix)#', "#@@ServerRoot@@##{opt_prefix}#"
```